src/xenia/vfs/devices/disc_image_device.cc

```diff
@@ -127,9 +127,11 @@ bool DiscImageDevice::ReadEntry(ParseState* state, const uint8_t* buffer,
   entry->attributes_ = attributes | kFileAttributeReadOnly;
   entry->size_ = length;
   entry->allocation_size_ = xe::round_up(length, bytes_per_sector());
-  entry->create_timestamp_ = 0;
-  entry->access_timestamp_ = 0;
-  entry->write_timestamp_ = 0;
+
+  // Set to January 1, 1601 (UTC) in 100-nanosecond intervals
```

Comment:
Nope.
If we track the usage of this data, it will end in the pair of functions NtQueryInformationFile and NtQueryFullAttributesFile, that have respective WinAPI originals, and they are using structure X_FILE_NETWORK_OPEN_INFORMATION, that just a copy of FILE_NETWORK_OPEN_INFORMATION, and if we check the [docs](https://msdn.microsoft.com/en-us/library/windows/hardware/ff545822(v=vs.85).aspx), we will read "100-nanosecond intervals since the start of the year 1601 in the Gregorian calendar.". 
Next, we can see that values are just copied:
```
auto file_info = file_info_ptr.as<X_FILE_NETWORK_OPEN_INFORMATION*>();
file_info->creation_time = file->entry()->create_timestamp();
```
And these two functions are only one who uses that information.
Therefore, there is no gain in recalculating it to Unix timestamps, and zero is definitely January 1, 1601. Also, filesystem_win.cc just stores values from FILETIME as it is, and it is 1601-based too.